images/ci-prune-buildcache/ci_buildcache_prune.py

```diff
@@ -495,7 +501,8 @@ def configure_parser():
         except Exception as e:
             print(f"Error -- Skipping pruning of {stack}")
             print(str(e))
-            raise "ff mode" from e
+            if args.fail_fast:
+                raise "fail fast" from e
```

Comment:
This `raise` statement was in the error stack trace as well, it came up in handling of the first exception.  As mentioned there, you can't `raise` a string because it doesn't derive from `BaseException`.

Reply:
interesting, I was not having issue with this locally, maybe the from makes it work?

Reply:
The `from` does not make it work. You must not have hit this code path locally, or else didn't notice the "During handling of the above exception..." bit.